docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -591,7 +591,8 @@ def test_uses_context():
 ## Examples
 
 - [Multi-component asset keys](#multi-component-asset-keys)
-- [Attaching metadata](#attaching-metadata)
+- [Recording materialization metadata](#attaching-materialization-metadata)
+- [Attaching definition metadata](#attaching-definition-level-metadata)
```

Comment:
```suggestion
- [Recording materialization metadata](#recording-materialization-metadata)
- [Attaching definition metadata](#attaching-definition-metadata)
```

These guys weren't working because the anchors don't match the headings. This should fix it.